ext/CMakeLists.txt

```diff
@@ -12,24 +12,38 @@ target_include_directories(ext-tclap INTERFACE tclap/include)
 # Alias system TBB or build locally and export that, according
 # to ARB_PRIVATE_TBBLIB setting.
 
-if(ARB_REQUIRE_TBB)
+find_package(TBB)
+if(ARB_PRIVATE_TBBLIB OR NOT TBB_FOUND)
```

Comment:
I''m not really concerned about the exact semantics here: there is more than one 'right way'. If you'd prefer that users must explicitly set the ARB_PRIVATE_TBBLIB option to use the included TBB, then that's also cool.

Reply:
I thought that the approach you were taking was to explicitly set `ARB_PRIVATE_TBBLIB`

Reply:
Well, the logic there kinda captures what I thought would be useful: if TBB is requested and we can't find it, or the user requests the in-source one, then we build the in-source one. I'm totally happy with the alternative, of it being an error if TBB isn't found and the explicit request is not made. Do you have a preference?

Reply:
What you have is fine then! I thought that your preference was for the in-source TBB to be strictly opt-in.
I actually prefer this approach to making it strictly opt-in, because I like using the in-source version, and don't feel like explicitly asking for it!